libtransact/src/workload/command.rs

```diff
@@ -75,24 +96,41 @@ impl TransactionHandler for CommandTransactionHandler {
     }
 }
 
-#[derive(Debug)]
-pub enum Command {
-    Set { address: String, value: Vec<u8> },
-    Delete { address: String },
-    Get { address: String },
-    Fail { error_msg: String },
-}
-
 impl std::fmt::Display for Command {
     fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
         match self {
-            Command::Set {
-                ref address,
+            Command::SetState {
```

Comment:
This display implementation is just for rendering these commands for the tests also replaced by the protobuf classes.